sqlserver/tests/compose-hc/setup.sh

```diff
@@ -0,0 +1,23 @@
+# Run the setup script to create the DB.
+# Do this in a loop because the timing for when the SQL instance is ready is indeterminate.
+for i in {1..120};
+do
+    /opt/mssql-tools/bin/sqlcmd -S localhost -U sa -P $SA_PASSWORD -d master -i is_running.sql -b
```

Comment:
Why not inline the query with `-q` command line instead of putting it in a separate file?